esmvaltool/diag_scripts/sea_surface_salinity/compare_salinity.py

```diff
@@ -0,0 +1,261 @@
+import logging
+import string
+import os
+from datetime import datetime
+from functools import reduce
+import matplotlib.pyplot as plt
+from matplotlib.legend_handler import HandlerBase
+from matplotlib.text import Text
+from matplotlib.legend import Legend
+import numpy as np
+import cf_units
+import iris
+from iris.util import unify_time_units
+import iris.quickplot as qplot
+
+from iris.analysis.stats import pearsonr
+from iris.coord_categorisation import add_month_number, add_year
+
+import esmvaltool.diag_scripts.shared
+import esmvaltool.diag_scripts.shared.names as n
```

Comment:
you either import one module as `module` and then use its attributes/properties as `module.attr` or import attrs/properties separately depending what you need, also please don't import a module as `n` - it's nondescript and usually single letters are not good not even for variables, let alone modules

Reply:
Thanks @valeriupredoi for fixing this!